turn/Dockerfile

```diff
@@ -0,0 +1,11 @@
+ARG VERSION
+FROM instrumentisto/coturn:${VERSION:-latest}
+
+RUN apk add --no-cache openssl
+
+ADD ./rootfs/defaults/docker-entrypoint.sh /docker-entrypoint.sh
+
+ENTRYPOINT ["/docker-entrypoint.sh"]
+
+EXPOSE 5349 8443 10000:11000/udp
+
```

Comment:
It's recommended to run containers without root. Can you please add a line like I did it in my OpenShift PR and test it, otherwise I have to change it in my PR. (FYI OpenShift will launch containers with random UIDs and all users are always member of the root group)
Would be great to have new image already ready to run as an unprivileged user.

```dockerfile
USER 1001
```

Reply:
Could you please make PR around it, when this PR will merged?

Reply:
That's fine for me because my PR is also not merged yet. I have to rebase it anyway if this PR becomes merged first which I think will be the case.